assets/scss/settings/_z-index.scss

```diff
@@ -0,0 +1,4 @@
+$z-index-base: 0;
+$z-index-low: 1;
+$z-index-mid-low: 2;
+$z-index-mid: 3;
```

Comment:
That's an interesting approach, but it may come in handy to have a few "component oriented" z-indexes e.g. `$index-header` or `$index-mega-mennu`. 

That way you could set the `z-index` relatively to a component, and won't need to worry about updating z-indexes in multiple places if you'll need to change some crucial z-index value